test/e2e/performanceprofile/functests/README.md

```diff
@@ -22,4 +22,27 @@ DISCOVERY_MODE: to get an already deployed performanceProfile.
 If DISCOVERY_MODE set to true the suites will search for a PerformanceProfile on the cluster and use it.
 If no PerformanceProfile is found, that suites will be skipped.
 
-RESERVED_CPU_SET, ISOLATED_CPU_SET, OFFLINED_CPU_SET: strings that present the CPU sets distributed between reserved, isolated, and offlined CPU profile specifications. The runner is responsible for validating that these values are compatible with the testing environment. 
\ No newline at end of file
+RESERVED_CPU_SET, ISOLATED_CPU_SET, OFFLINED_CPU_SET: strings that present the CPU sets distributed between reserved, isolated, and offlined CPU profile specifications. The runner is responsible for validating that these values are compatible with the testing environment.
+
+### Running Tests Locally on Hypershift Platform
+
+When running e2e tests locally on a Hypershift platform, there are a couple of Environment variables 
+that should be set in advanced:
+
+* `CLUSTER_TYPE=hypershift` - A const that used to configure the e2e tests for running on Hypershift.
+
+
+* `CLUSTER_NAME` - Represents the name of the hosted-cluster. 
+When running locally,
+the name is usually the output of  `oc get hc <name-of-your-hosted-cluster> -n clusters --no-headers -o custom-columns=NAME:.metadata.name`
+
+
+* `HYPERSHIFT_MANAGEMENT_CLUSTER_KUBECONFIG` - The path for the Kubeconfig file of the management/hub cluster.
+
+
+* `KUBECONFIG` - The path for the Kubeconfig file of the hosted cluster.
+
+
+* `HYPERSHIFT_HOSTED_CONTROL_PLANE_NAMESPACE` - The name of the namespace where all the control-plane components reside.
```

Comment:
I strongly suggest replacing:
```
<key> - <key_explanation>
```
in favour of
```
<key>: <key_explanation>
```
Note there is no space between `key` and `:`.  This is also what OpenShift docs seem to be using.  Using a hyphen for this is not a correct usage in my view.